rpcs3/Emu/RSX/VK/VKGSRender.cpp

```diff
@@ -2273,6 +2273,10 @@ void VKGSRender::do_local_task(rsx::FIFO_state state)
 		// NOTE: This may cause graphics corruption due to unsynchronized modification
 		on_invalidate_memory_range(m_offloader_fault_range, m_offloader_fault_cause);
 		m_queue_status.clear(flush_queue_state::deadlock);
+
+		// Abort all other operations, this is likely coming from offloader::sync() and we need to unwind the call stack.
+		// If flush_command_queue is executed at this point, recursion will cause a deadlock will occur.
```

Comment:
Wording

Reply:
Maybe suggest something, looks good to me.

Reply:
... recursion will cause a deadlock will occur
To
... recursion will cause a deadlock to occur
Or something like that

Reply:
ok, nice catch. ~~msvc grammarly extension needed~~

Reply:
:man_student: